Week1/credit/credit.c

```diff
@@ -0,0 +1,85 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    // input creditcard number
+    long n = get_long("Creditcardnumber: ");
+
+    // Count lenght of Creditcardnumber
+    int i = 0;
+    long cc = n;
+    while (cc > 0)
+    {
+        cc = cc / 10;
+        i++;
+    }
+
+    // Check the lenght
+    if (i != 13 && i != 15 && i != 16)
+    {
+        printf("INVALID\n");
+        return 0;
+    }
+
+    // Calculate checksum
+    int sum1 = 0;
+    int sum2 = 0;
+    long x = n;
+    int total = 0;
+    int mod1;
+    int mod2;
+    int d1;
+    int d2;
+    do
+    {
+        // Remove last digit and add to sum1
```

Comment:
Remove comments :)

Try to change the name of the variables in something that describes what the variable does.
If you do that well enough you most likely wont need the comments anymore :)